resources/config.toml

```diff
@@ -26,6 +26,12 @@ default_locale='fr-fr'
   PORT = 8889  # will be lower cased when reading.
   base_url = "http://localhost:8889/"
   timeout = 2
+  telegram_bot = "test_test_icu_bot"
+  telegram_updates_every = 10
+  [messaging.prod]
+  PORT = 8889  # will be lower cased when reading.
+  base_url = "http://localhost:8889/"
+  timeout = 2
```

Comment:
The above 3 lines can probably be removed? Since we no longer use dev/prod setup?

Reply:
Above comment is still relevant.